src/dash-adapter.js

```diff
@@ -1297,7 +1299,14 @@ export default class DashAdapter extends BaseMediaSourceAdapter {
    */
   getStartTimeOfDvrWindow(): number {
     if (this.isLive() && this._shaka) {
-      return this._shaka.seekRange().start + this._shaka.getConfiguration().streaming.safeSeekOffset;
+      if (!this._isStartOver) {
+        if (this._seekRangeStart > this._shaka.seekRange().start) {
+          // seekRange().start seeked back means this is start over
```

Comment:
so this will only happen for segment list manifests, correct? and also only if api is called in times where condition `this._seekRangeStart > this._shaka.seekRange().start` is true which doesn't happen all the time.
I think if this is the case we can still leave the calculation in manifest request where this condition will always be correct for segment list manifest

Reply:
on manifest request `this._shaka.seekRange().start` is not updated yet